index.html

```diff
@@ -94,6 +97,7 @@ <h2 class="authorClass">Author</h2>
 <footer>
   <section id="description-total">
     <dt> Total Words in Project Descriptions:</dt> <dd class="words"></dd>
+    <!-- Right now I have a number counter that is counting the number of words in the project descriptions. This was for functionality purposes but I would like to incorporate some stats that are meaningful to my site. Since my portfolio is a record of all the things I have worked on I think that I would like to track something more relateable. I was thinking about adding a category to my page that was either hours spent on each project or skills used. My stats could then be total # of skills used or total # of hrs spent on all projects. -->
```

Comment:
Great job having in mind how to adapt the counter. I think a lot of us weren't quite sure how to adapt it to our Portfolio at first, and first incorporated it in a way similar to the lab. I like your idea about using this functionality for time spent, or counting the skills per project to showcase your abilities that you used for that project.